content/ri/installing/install-k8s.md

```diff
@@ -9,210 +9,174 @@ nextStep:
-```
+1. Once the deployment and service are successfully applied and complete, access RedisInsight. You use the `<external-ip>` of the service you created to reach RedisInsight.
 
-{{< note >}}
-The service type is [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) which allows us to access redisinsight from outside k8s cluster.
-{{< /note >}}
+    ```sh
+    $ kubectl get svc redisinsight-service
+    NAME                   CLUSTER-IP       EXTERNAL-IP      PORT(S)         AGE
+    redisinsight-service   <cluster-ip>     <external-ip>    80:32143/TCP    1m
+    ```
 
-You get the following output:
-```
-NAME: redisinsight
-LAST DEPLOYED: Wed Dec 16 10:46:08 2020
-NAMESPACE: default
-STATUS: deployed
-REVISION: 1
-NOTES:
-1. Get the application URL by running these commands:
-  export NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services redisinsight-redisinsight-chart)
-  export NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
-  echo http://$NODE_IP:$NODE_PORT
+## Create a persistent volume claim
 
-```
+For a deployment with persistent writeable volumes, add a `PersistentVolumeClaim` to your `redisinsight.yaml` file. A few changes to the deployment are also necessary. The example below creates a persistent volume created from a volume claim template.
 
-2. Run the commands mentioned in the output to get the end point:
+1. Make the following updates to your deployment spec:
 
-```sh
-export NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services redisinsight-redisinsight-chart)
-export NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
-echo http://$NODE_IP:$NODE_PORT #Example: http://172.17.0.2:32388
+    a. Set the replica strategy (spec.replicas.strategy) `Recreate`. This will avoid multiple pods trying to use the same volume.
 
-```
+      ```yaml
+      strategy:
+      type: Recreate
+      ```
+  
+    b. In the `template.spec.volumes` section, replace the `emptyDir` volume with your persistent volume claim.
+  
+      ```yaml
+      volumes:
+          - name: db
+            persistentVolumeClaim:
+              claimName: redisinsight-pv-claim
+      ```
+  
+    c. Add an  init container (`spec.initContainers`) to configure write access to the container.
+
+      ```yaml
+        initContainers:
+          - name: init
+            image: busybox
+            command:
+              - /bin/sh
+              - '-c'
+              - |
+                chown -R 1001 /db
+            resources: {}
+            volumeMounts:
+              - name: db
+                mountPath: /db
+            terminationMessagePath: /dev/termination-log
+            terminationMessagePolicy: File
+      ```
+
+1. Add the following persistent volume claim to your deployment file (`redisinsight.yaml`).
```

Comment:
May want to add something like "you may want to change the "default" storageClassName to your preferred storage class.

Reply:
+1 - the storage class name is key here and has to be a conscious choice from the user. Plus there might not be a default storage class.